.gitignore

```diff
@@ -5,3 +5,4 @@
 vendor/*/
 
 test-reports/
+elsy_build_logs/
```

Comment:
Could we just call the directory `build_logs`? We've really only been using `elsy` for searchability.